### PR TITLE
Small refactor to reduce LOC and keep code DRY.

### PR DESCRIPTION
**Pull request by V0idMatr1x**
All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally before submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -4,16 +4,18 @@ import (
...
 
 	log "github.com/sirupsen/logrus"
 )
 
-var quit = make(chan struct{})
-
 func main() {
-	var configurationsCorePath string
-	var configurationsServicesDirectory string
+	// removed unnecessary fmt.Sprintf call
```
Comment (mariocandela) [inline main.go:12]: I really appreciate your work ❤️  I will approve it once you remove all comments

File: main.go [hunk 2]
```diff
@@ -4,16 +4,18 @@ import (
...
+	// consolidated variables into a single declaration to reduce loc and keep DRY
```
Comment (mariocandela) [inline main.go:13]:
```suggestion
```

File: tracer/tracer.go [hunk 3]
```diff
@@ -105,12 +106,10 @@ func (tracer *tracer) TraceEvent(event Event) {
 	switch event.Protocol {
 	case HTTP.String():
 		eventsHTTPTotal.Inc()
-		break
 	case SSH.String():
 		eventsSSHTotal.Inc()
-		break
 	case TCP.String():
+		// Removed redundant case breaks
```
Comment (mariocandela) [inline tracer/tracer.go:112]:
```suggestion
```

File: plugins/openai-gpt.go [hunk 4]
```diff
@@ -62,7 +65,6 @@ type gptRequest struct {
 }
 
 // Reference: https://www.engraved.blog/building-a-virtual-machine-inside/
```
Comment (mariocandela) [inline plugins/openai-gpt.go:67]: Please move up 'promptVirtualizeLinuxTerminal' declaration.

File: main.go (hunk 1 above)
Comment (mariocandela) [inline main.go:12]:
```suggestion
```

**Review by mariocandela — COMMENTED** [review]
I really appreciate your work ❤️ I will approve it once you remove all comments

**Comment by V0idMatr1x** [comment]
> I appreciate your work ❤️ I will approve it once you remove all comments

I appreciate you letting me contribute to your project! 
Thank you for your time and the code review!

I hope to make more meaningful contributions in the future 😄

File: plugins/openai-gpt.go [hunk 5]
```diff
@@ -5,19 +5,22 @@ import (
...
 	log "github.com/sirupsen/logrus"
-
+	
 	"github.com/go-resty/resty/v2"
 )
 
-const ChatGPTPluginName = "OpenAIGPTLinuxTerminal"
-const openAIGPTEndpoint = "https://api.openai.com/v1/completions"
+const (
+	promptVirtualizeLinuxTerminal = "I want you to act as a Linux terminal. I will type commands and you will reply with what the terminal should show. I want you to only reply with the terminal output inside one unique code block, and nothing else. Do no write explanations. Do not type commands unless I instruct you to do so.\n\nA:pwd\n\nQ:/home/user\n\n"
```
Comment (mariocandela) [inline plugins/openai-gpt.go:15]:
```suggestion
	// Reference: https://www.engraved.blog/building-a-virtual-machine-inside/
	promptVirtualizeLinuxTerminal = "I want you to act as a Linux terminal. I will type commands and you will reply with what the terminal should show. I want you to only reply with the terminal output inside one unique code block, and nothing else. Do no write explanations. Do not type commands unless I instruct you to do so.\n\nA:pwd\n\nQ:/home/user\n\n"
```

**Review by mariocandela — CHANGES_REQUESTED** [review]
For any future changes or ideas, kindly initiate a discussion on the following GitHub page: https://github.com/mariocandela/beelzebub/discussions :smile:
Thanks for your time mate

**Comment by V0idMatr1x** [comment]
> For any future changes or ideas, kindly initiate a discussion on the following GitHub page: https://github.com/mariocandela/beelzebub/discussions 😄 Thanks for your time mate

Will do! You are very welcome!

**Review by mariocandela — APPROVED** [review]
LGTM

**Pull request merged**